quinn-proto/src/connection/mod.rs

```diff
@@ -2363,7 +2363,7 @@ where
     fn populate_packet(
         &mut self,
         space_id: SpaceId,
-        minimum_length: usize,
+        min_size: usize,
```

Comment:
Can we move this part into the previous commit?

Reply:
Done.